### ISSUE TITLE
Re-Enable Lottie Support for Windows

### ISSUE THREAD

**Issue by strseb** [mozilla-mobile/mozilla-vpn-client#2509]
The Project currently does not build on windows, when lottie is enabled 
Its seems qmake generates an invalid vcxproj.

The CI mistakenly does not catch this and reports a success even though the logs say: 
```
Build started 1/3/2022 2:59:00 PM.
Project "D:\a\mozilla-vpn-client\mozilla-vpn-client\MozillaVPN.vcxproj" on node 1 (Clean target(s)).
D:\a\mozilla-vpn-client\mozilla-vpn-client\MozillaVPN.vcxproj(2888,5): error MSB4035: The required attribute "Include" is empty or missing from the element <None>.
Done Building Project "D:\a\mozilla-vpn-client\mozilla-vpn-client\MozillaVPN.vcxproj" (Clean target(s)) -- FAILED.

Build FAILED.

"D:\a\mozilla-vpn-client\mozilla-vpn-client\MozillaVPN.vcxproj" (Clean target) (1) ->
  D:\a\mozilla-vpn-client\mozilla-vpn-client\MozillaVPN.vcxproj(2888,5): error MSB4035: The required attribute "Include" is empty or missing from the element <None>.

    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:00.05
Failed to clean up the project
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-1502)

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Andrea Marchesini Brian Yee Sebastian Streich This as a blocker for QA team in regards of testing the v2.7 Mozilla VPN last build, on Windows. Would be great if this gets fixed by tomorrow.

Thank you!

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Thanks for the fix! Please add the fix in the main or release 2.7 branch as well.

**Comment by data-sync-user** [comment]
➤ Andrea Marchesini commented:

Fix in VPN-1544
